src/saltext/salt_describe/modules/salt_describe_cron.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Module for building state file
+
+.. versionadded:: 3006
```

Comment:
This should probably be 1.1.0 to follow salt-describe's versioning. 

Same goes for all versionadded in this PR